.github/workflows/tutorials.yml

```diff
@@ -0,0 +1,52 @@
+name: Jupyter notebooks
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Test Jupyter notebooks on ${{ matrix.python-version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.7']
+        os: [ubuntu-latest]
+        gcc-version: [9]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install GCC ${{ matrix.gcc-version }}
+      run : |
+        sudo apt-get install -y g++-${{ matrix.gcc-version }}
+
+    - name: Install MPI
+      run : |
+        sudo apt-get install -y mpich libmpich-dev
+
+    - name: Install dependencies
+      run: |
+        pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install -r requirements-optional.txt
+        pip install -e .[extras]
```

Comment:
`pip install -e .[extras]` does the same as the two `requirements.txt` + `requirements-optional.txt` so only need that one

Reply:
Resolved in [`2895d17`](https://github.com/devitocodes/devito/pull/1069/commits/2895d17ce81d1cc1ef04e9c1e7d2e76dacc081e9)